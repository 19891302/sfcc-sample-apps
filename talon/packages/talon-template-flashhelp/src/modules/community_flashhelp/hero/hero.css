.jumbotron-fixed-height-wrapper {
    width: 100%;
    margin-bottom: 0;
    padding-bottom: 0;
    position: relative;
    display: block;
}

.jumbotron {
    font-size: 22px;
    line-height: 1.27;
    font-weight: 300;
    position: relative;
    height: 700px;
}

.jumbotron.harvey {
    background-image: linear-gradient(to bottom, rgba(0,0,0,0.2), rgba(0,0,0,0.2)), var(--lwc-header-image);
    background-size: cover;
    background-position: top;
}

.jumbotron .text-align-center {
    margin: 0 0 18px;
}

.jumbotron .text-block {
    height: auto;
    position: absolute;
    bottom: 64px;
    padding: 0 20px;
    right: 0;
    left: 0;
    font-size: 22px;
    line-height: 1.182em;
    font-weight: 200;
    color: #fff;
}

.jumbotron .page-subtitle {
    line-height: 50px;
    font-size: 30px;
}

.jumbotron .color-subdued {
    color: #e8e8e8;
}

.jumbotron h1 {
    margin: 0 0 18px;
}

@media (min-width: 1023px) {
    .jumbotron h1 {
        font-size: 50px;
    }
}

@media (min-width: 1365px) {
    .jumbotron h1 {
        font-size: 60px;
    }
}

