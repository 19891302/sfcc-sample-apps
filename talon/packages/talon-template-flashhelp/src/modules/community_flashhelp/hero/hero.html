<template>
    <div class="jumbotron harvey">
        <div class="container-fluid slds-text-align_center text-block">
            <h1 class="hidden-sm hidden-xs">Hurricane Harvey Relief Information</h1>
                <p class="page-subtitle color-subdued">Across storm-battered areas, we continue to deliver support</p>
        </div>
    </div>
</template>