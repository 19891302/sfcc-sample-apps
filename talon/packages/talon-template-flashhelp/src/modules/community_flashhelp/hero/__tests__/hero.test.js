import { createElement } from 'lwc';
import hero from 'community_flashhelp/hero';

describe('community_flashhelp/hero', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-hero', { is: hero });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});
