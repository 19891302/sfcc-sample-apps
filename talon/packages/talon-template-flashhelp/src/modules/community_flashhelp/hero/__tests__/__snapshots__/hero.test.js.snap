// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/hero renders correctly 1`] = `
<community_flashhelp-hero>
  #shadow-root(open)
    <div
      class="jumbotron harvey"
    >
      <div
        class="container-fluid slds-text-align_center text-block"
      >
        <h1
          class="hidden-sm hidden-xs"
        >
          Hurricane Harvey Relief Information
        </h1>
        <p
          class="page-subtitle color-subdued"
        >
          Across storm-battered areas, we continue to deliver support
        </p>
      </div>
    </div>
</community_flashhelp-hero>
`;
